src/config.rs

```diff
@@ -80,10 +80,28 @@ impl Config {
         &self,
         existing_ref_names: &HashSet<String>,
         title: &str,
+    ) -> String {
+        self.find_unused_branch_name(&existing_ref_names, &slugify(title))
+    }
+
+    pub fn get_base_branch_name(
+        &self,
+        existing_ref_names: &HashSet<String>,
+        title: &str,
+    ) -> String {
+        self.find_unused_branch_name(
+            &existing_ref_names,
+            &format!("{}/{}", &self.master_branch, &slugify(title)),
```

Comment:
I like having the master branch name in there, since it makes it clear that it's a stand-in for that branch (whereas "base" might not be clear to some people), and I thought it was nice having it as a prefix since it's easier to miss a suffix if you have "Merging 1 commit into `spr/flooey/really-long-branch-name-thing.base` from `spr/flooey/really-long-branch-name-thing`", so I went with `format!("{}.{}", master_branch, title)`